src/main/java/org/broadinstitute/hellbender/tools/funcotator/DataSourceFuncotationFactory.java

```diff
@@ -29,5 +72,19 @@ default void close() {}
      * @param featureList {@link List} of {@link Feature} corresponding to the given {@code variant}.
      * @return {@link List} of {@link Funcotation} given the {@code variant}, {@code referenceContext}, and {@code featureContext}.  This should never be empty.
      */
-    List<Funcotation> createFuncotations(final VariantContext variant, final ReferenceContext referenceContext, final List<Feature> featureList);
+    public abstract List<Funcotation> createFuncotations(final VariantContext variant, final ReferenceContext referenceContext, final List<Feature> featureList);
```

Comment:
Currently the required ordering is Gencode first, then anything else.

I made those two methods so that if a datasource didn't need Gencode it could still be called to create `Funcotations` on a variant.

Reply:
Sounds good.